### PR TITLE
README update

### PR DESCRIPTION
**Pull request by varon**
Updates the Readme to be more clear, shorter (so we can list it on the Nuget.org packages).

Opinions, thoughts?

### CODE REVIEW

**Comment by VPeruS** [comment]
Auto generated changelogs based on prs and it's tags

File: README.md [hunk 1]
```diff
@@ -186,120 +202,15 @@ Your favorite IDE will display inline documentation for all OpenTK APIs. Additio
 
 Technical documentation about the implementation of OpenTK can be found in the [Technical Wiki](https://github.com/opentk/opentk/wiki).
 
-
 Need Help?
 ==========
 
-Post your questions at the [OpenTK forums](http://www.opentk.com/forum).
+Come chat with us on [Discord](https://discord.gg/GZTYR4s).
```
Comment (VPeruS) [inline README.md]: It's already in first lines

Comment (varon) [reply README.md]: They're different; the top says "join discord to help us." the bottom says "join discord if you need help."

File: README.md [hunk 2]
```diff
@@ -34,6 +29,9 @@ Features
 - Create cutting-edge graphics with OpenGL 4.6 and OpenGL ES 3.0
 - Spice up your GUI with 3d acceleration
 - Improve your code flow with strong types and inline documentation
+- Windowing systems to help get you started
+- Input, and other game essentials.
+- Performant, highly optimized and bug-free linear algebra library
```
Comment (Nihlus) [inline README.md]: Bug-free is a dangerous term lol

**Comment by Nihlus** [comment]
Looks good!

File: README.md (hunk 2 above)
Comment (varon) [reply README.md]: Will do

**Comment by DylanFPS** [comment]
Woah sorry I closed this, didn't mean to :P

**Pull request merged**